src/at2/Man.java

```diff
@@ -0,0 +1,31 @@
+package at2;
+
+public class Man extends Human {
+    public final String sex = "male";
+    private boolean beard;
+
+    Man() {};
+
+    Man(String name, int age, double height, double weight, String hairColor) {
+        super(name, age, height, weight, hairColor);
+    }
+
+    public String getSex() {
+        return "male";

```

Comment:
why do you return hardcoded value in case if you have dedicated 'sex' variable to store it?